### ISSUE TITLE
Log into a server/ grouping

### ISSUE THREAD

**Issue by IeuanWalker** [tareqimbasher/NetPad#101]
Any plans to allow log into a server, rather than just a specific DB?
Then just perform the scaffolding when a database within it is selected?

We have a lot of databases, so it's quiet time censusing setting up a new connection for each one. 
Also gets quite messy with all the tests and prod database together without the ability sort things in folders.

Azure data studio does a pretty good job at it - 
![image](https://github.com/tareqimbasher/NetPad/assets/6544051/cbf80fee-bc17-4b21-aed3-7db01cb99b8e)

**Comment by tareqimbasher** [comment]
Yes I do plan on adding that ability. Don't have an estimate on when just yet. I like the layout I'll keep it in mind, thank you for the feedback!

**Comment by tareqimbasher** [comment]
This is now complete and will go out in the upcoming release. Users can now add servers and select databases on that server that will share the connection and scaffolding options set on the server. Next, I will be adding folders for more organization as requested in #31, which would give you the ability to move servers or standalone database connections into folders.

**Comment by tareqimbasher** [comment]
A new beta build [v0.12.0-beta.1](https://github.com/tareqimbasher/NetPad/releases/tag/v0.12.0-beta.1) is out that includes the feature. A full release will follow in the next couple days.

**Comment by tareqimbasher** [comment]
[v0.12.0](https://github.com/tareqimbasher/NetPad/releases/tag/v0.12.0) is now available with the ability to add servers!
